cli/teos_cli.py

```diff
@@ -228,35 +229,24 @@ def get_all_appointments(teos_url):
         return None
 
 
-def load_keys(teos_pk_path, user_sk_path):
+def load_keys(user_sk_path):
     """
-    Loads all the keys required to sign, send, and verify the appointment.
+    Loads all the user private key and id.
 
     Args:
-        teos_pk_path (:obj:`str`): path to the tower's public key file.
         user_sk_path (:obj:`str`): path to the user's private key file.
 
     Returns:
-        :obj:`tuple`: a three-item tuple containing a ``str``, a ``PrivateKey`` and a ``str``
-        representing the tower id (compressed pk), user sk and user id (compressed pk) respectively.
+        :obj:`tuple`: a tuple containing a ``PrivateKey`` and a ``str`` representing the user sk and user id
+        (compressed pk) respectively.
 
     Raises:
         :obj:`InvalidKey <cli.exceptions.InvalidKey>`: if any of the keys is invalid or cannot be loaded.
```

Comment:
Remove `any of` from the docs, as there's now only one key in this function.

Reply:
Actually it loads both the private and public key, so any applies here.